### DISCUSSION TITLE
Ability to add animation tracks on uneditable imported animations

### DISCUSSION THREAD

**Discussion by Tormentous** [godotengine/godot-proposals#10192] [Editor]
Now I don't mean rotation transform or scale animation tracks I mean the ones where they decrease the need for code, for example I import a new model and I want to add sound effects or a call function to an animation without having to make unique or go through that whole process like if I was going to completely overhaul the animation

**Comment by theraot** [comment]
I insist we need animation inheritance.

See:
- https://github.com/godotengine/godot-proposals/issues/9802
- https://github.com/godotengine/godot-proposals/issues/3858
